app/_src/policies/meshfaultinjection.md

```diff
@@ -0,0 +1,263 @@
+---
+title: MeshFaultInjection (beta)
+---
+
+With the MeshFaultInjection policy you can easily test your microservices against resiliency.
+
+{% warning %}
+This policy uses a new policy matching algorithm and is in beta state. It should not be combined with [FaultInjection](/docs/{{ page.version }}/policies/fault-injection).
+{% endwarning %}
+
+## `targetRef` support matrix
+
+| `targetRef.kind`    | top level | to  | from |
+| ------------------- | --------- | --- | ---- |
+| `Mesh`              | ✅        | ❌  | ✅   |
+| `MeshSubset`        | ✅        | ❌  | ✅   |
+| `MeshService`       | ✅        | ❌  | ✅   |
+| `MeshServiceSubset` | ✅        | ❌  | ✅   |
+
+To learn more about the information in this table, see the [matching docs](/docs/{{ page.version }}/policies/targetref).
+
+## Configuration
+
+`MeshFaultInjection` allows configuring a list of HTTP faults. They execute in the same order as they were defined.
+
+```yaml
+default:
+ http:
+   - abort:
+       httpStatus: 500
+       percentage: "2.5"
+     delay:
+       value: 5s
+       percentage: 5
+     responseBandwidth:
+       limit: "50mbps"
+       percentage: 50
+   - abort:
+       httpStatus: 500
+       percentage: 10
+   - delay:
+       value: 5s
+       percentage: 5
+```
+
+It's worth mentioning that percentage of the next filter depends on the percentage of previous ones.
```

Comment:
Is it because of how envoy works?

Reply:
Yes, that is because filters are applied sequentially, so Filter1 -> Filter2 -> ... -> FilterN and they have the same header match